sha256sum: add --ignore-missing example (#7929)
tldr-pages/tldr: pages/common/sha256sum.md

> - Only show a message for files for which verification fails:

Reviewer: ```suggestion
- Only show a message for files for which verification fails or missing files:
```

`sha256sum --check --quiet {{path/to/file.sha256}}`

> - Only show a message for files that are there and for which verification fails:

Reviewer: I think this is more clear for non-native English speakers
```suggestion
- Only show a message for files for which verification fails ignoring missing files:
```

Author: Well, I'm a non-native English speaker ;) And I agree with you. I didn't like my initial wording _"that are there"_ but struggled to come up with a better phrase. I think your edit is better but I want to add a comma after _"fails"_.

**Only show a message for files for which verification fails, ignoring missing files:**

Reviewer: sure
